cognite/client/utils/_auxiliary.py

```diff
@@ -19,6 +19,7 @@
 
 import cognite.client
 from cognite.client import utils
+from cognite.client.data_classes import LabelFilter
```

Comment:
is this used for anything?

Reply:
Not sure how it got added here.

Reply:
Ah, I was I think doing some experimentation.